Formula/fastlane.rb

```diff
@@ -27,14 +27,11 @@ def install
     system "gem", "build", "fastlane.gemspec"
     system "gem", "install", "fastlane-#{version}.gem", "--no-document"
 
-    (bin/"fastlane").write <<~EOS
-      #!/bin/bash
-      export PATH="#{Formula["ruby@2.7"].opt_bin}:#{libexec}/bin:$PATH"
-      export FASTLANE_INSTALLED_VIA_HOMEBREW="true"
-      GEM_HOME="#{libexec}" GEM_PATH="#{libexec}" \\
-        exec "#{libexec}/bin/fastlane" "$@"
-    EOS
-    chmod "+x", bin/"fastlane"
+    (bin/"fastlane").write_env_script libexec/"bin/fastlane",
+      PATH:                            "#{Formula["ruby@2.7"].opt_bin}:#{libexec}/bin:$PATH"
+      FASTLANE_INSTALLED_VIA_HOMEBREW: "true"
+      GEM_HOME:                        "#{libexec}"
+      GEM_PATH:                        "#{libexec}"
```

Comment:
```suggestion
      PATH:                            "#{Formula["ruby@2.7"].opt_bin}:#{libexec}/bin:$PATH",
      FASTLANE_INSTALLED_VIA_HOMEBREW: "true",
      GEM_HOME:                        libexec,
      GEM_PATH:                        libexec,
```
You might need to wrap the whole thing in `{ }` and make it `libexec.to_s` instead

Reply:
Sorry, I pushed an older version 🤦‍♂️ 

Making a force push now which is the version that is actually tested, and which uses `to_s` 😅